client/mass/about.ts

```diff
@@ -0,0 +1,39 @@
+import { getCompInit } from '../rx'
+import { Elem } from '../types/d3'
+import { MassAppApi } from './types/mass'
+
+type AboutObj = {
+	html: string
+}
+
+type MassAboutOpts = {
+	app: MassAppApi
+	holder: Elem
+	features: AboutObj
+}
+
+class MassAbout {
+	type: string
+	app: MassAppApi
+	holder: Elem
+	features: AboutObj
+
+	constructor(opts: MassAboutOpts) {
+		this.type = 'about'
+		this.app = opts.app
+		this.holder = opts.holder
+		this.features = opts.features
+	}
+
+	init() {
+		this.holder.append('div').style('padding', '10px').html(this.features.html)
+	}
+
+	main() {
+		const aboutIdx = this.app.Inner.components.nav.Inner.tabs.findIndex(tab => tab.subheader == 'about')
```

Comment:
@creilly8 and all, `Inner` is only present when debugmode flag is set which should only be on dev. it breaks prod now.
as a rule, Inner must not be used in portal code